index.js

```diff
@@ -46,8 +43,9 @@ module.exports = {
   },
 
   config(env, baseConfig) {
-    if (!env)
+    if (!env) {
```

Comment:
I can't really remember, why I introduce this guard. Maybe to eliminate any possibility of being the cause of a bug in early days. I will remove this in the next release.